src/desktop/apps/gallery_partnerships/templates/index.jade

```diff
@@ -284,3 +284,5 @@ block body
               });
 
           .gallery-partnerships2__form__contact Call us: +1 800 589-1350
+  script
+    include ../client/intercom.js
```

Comment:
Let's not include the code snippet for logged-in admins and when Intercom in not enabled/configured?

Reply:
Yep, that also sounds reasonable. What do you mean by Intercom not being enabled though? That seems like that'd be controlled on the Intercom end. Unless you mean testing for the `INTERCOM_APP_ID` environment variable or having another variable to control whether it's enabled?

Reply:
Yes. I think checking `INTERCOM_APP_ID` or even `INTERCOM_APP_ID && INTERCOM_ENABLED` would be useful, so we can have more control.